.cirrus.yml

```
@@ -0,0 +1,13 @@
+task:
+  name: flake check
+  container:
+    image: nixos/nix:latest
+  check_script:
+    - nix flake check
+
+task:
```

YAMLlint (1.38.0)

[error] duplication of key "task" in mapping

(key-duplicates)